enclave-runtime/src/tests.rs

```diff
@@ -526,20 +562,20 @@ fn test_top_pool() -> TestTopPool {
 }
 
 /// Decrypt `encrypted` and decode it into `StatePayload`
-fn state_payload_from_encrypted(encrypted: &[u8]) -> StatePayload {
+pub fn state_payload_from_encrypted(encrypted: &[u8]) -> StatePayload {
```

Comment:
Why did we make these public, are we using them in other files as well? Then I think we should move them to the 'fixtures' folder, so they can be re-used. This file here is a top-level module, nothing should depend on it.

Reply:
Because that'd be a refactoring change in an already huge PR. Can we agree do that in a separate issue?

Reply:
Alright 😌

Reply:
Resolving this conversation in favor of this isuse: https://github.com/integritee-network/worker/issues/560